app/models/throttle.rb

```diff
@@ -29,7 +29,7 @@ class Throttle < ApplicationRecord
       attempt_window: (AppConfig.env.reset_password_email_window_in_minutes || 60).to_i,
     },
     idv_resolution: {
-      max_attempts: (AppConfig.env.idv_max_attempts || 3).to_i,
+      max_attempts: (AppConfig.env.idv_max_attempts || 5).to_i,
```

Comment:
Not entirely relevant for the pull request, but do we need the fallbacks here, or could it be safe enough to always use the configuration value? Asking because I feel this would make future updates easier if we could reference the configuration value in the specs, rather than hard-coding them.

e.g.

```diff
- 4.times do
+ (AppConfig.env.idv_max_attempts.to_i - 1).times do
```

Reply:
we can and probably should make it a required configuration, yeah

Reply:
willfix

Reply:
Actually, there's a few that fit into this category I think. I will address those in a follow on

Reply:
PR to make this change: https://github.com/18F/identity-idp/pull/4738